core/repository/sparql/src/main/java/org/eclipse/rdf4j/repository/sparql/SPARQLConnection.java

```diff
@@ -938,9 +938,9 @@ private void createBGP(StringBuilder qb, Resource subject, IRI predicate, Value
 				qb.append(SPARQLUtil.encodeString(lit.getLabel()));
 				qb.append("\"");
 
-				if (Literals.isLanguageLiteral(lit)) {
+				if (lit.getLanguage().isPresent()) {
```

Comment:
Arbitrary change this - both methods do exactly the same thing. We should perhaps look into deprecating the util method.